APIs/OData.qmd

```diff
@@ -17,6 +17,8 @@ import pandas as pd
 
 OData (Open Data Protocol) is a standard that specifies a variety of best practices for creating and using REST APIs. OData makes it possible to build REST-based data services that let Web clients publish and edit resources that are recognized by Uniform Resource Locators (URLs) and described in a data model using straightforward HTTP messages.
 
+# OData Products endpoint
```

Comment:
```suggestion
## OData Products endpoint
```

Reply:
Since the h1 would be the title set for the qmd so it is suggested that the other sub-heading startes with h2 i.e ## in markdown and so on for the sub-sub-headings